docs/calico-with-docker/docker-network-plugin/CalicoAsService.md

```diff
@@ -0,0 +1,72 @@
+<!--- master only -->
+> ![warning](../../images/warning.png) This document applies to the HEAD of the calico-containers source tree.
+>
+> View the calico-containers documentation for the latest release [here](https://github.com/projectcalico/calico-containers/blob/v0.14.0/README.md).
+<!--- else
+> You are viewing the calico-containers documentation for release **release**.
+<!--- end of master only -->
+
```

Comment:
Overall concern that someone coming to this document won't really get what it's about.
-  Perhaps the focus of this should be "How to run the Calico node containers as a service" or something similar.  That way you can describer how to do it for calico node and what else you need to do if you want to use Calico as a Docker network pluging.

Also, where is this linked from?